.github/dependabot.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    open-pull-requests-limit: 3
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+  - package-ecosystem: pip
+    directory: "/cellxgene_schema_cli/"
+    open-pull-requests-limit: 3
+    schedule:
+      interval: "daily"
```

Comment:
can we relax the interval to monthly? since we release relatively infrequently, once a month max

Reply:
that sounds reasonable. I'll also increase the number of PR is can open to infinite. Around migration time we can sift through them around migration time.